docs/developer-docs/smart-contracts/call/arguments.mdx

```diff
@@ -145,22 +149,11 @@ class Day(Variant, total=False):
     Fri: str
     Sat: str
 
+
 @query
-def get_text(d: Day) -> str:
-    if d == Day.Sun:
-        return "Sunday"
-    elif d == Day.Mon:
-        return "Monday"
-    elif d == Day.Tue:
-        return "Tuesday"
-    elif d == Day.Wed:
-        return "Wednesday"
-    elif d == Day.Thu:
-        return "Thursday"
-    elif d == Day.Fri:
-        return "Friday"
-    elif d == Day.Sat:
-        return "Saturday"
+def keyword_variant(variant_keyword: Day) -> Day:
```

Comment:
I would expect this example to look more like the other, where there is a switch or if statement to return something based on a variant...it also might be good for these examples to show how to create a variant.

Reply:
Revised per feedback